pkg/sync/issues.go

```diff
@@ -167,33 +163,33 @@ func UpdateIssue(config cfg.Config, ghIssue github.Issue, jIssue jira.Issue, ghC
 
 // CreateIssue generates a JIRA issue from the various fields on the given GitHub issue, then
 // sends it to the JIRA API.
-func CreateIssue(config cfg.Config, issue github.Issue, ghClient clients.GitHubClient, jClient clients.JIRAClient) error {
-	log := config.GetLogger()
+func CreateIssue(cfg config.Config, issue github.Issue, ghClient ghClient.GitHubClient, jClient jClient.JIRAClient) error {
+	log := cfg.GetLogger()
 
 	log.Debugf("Creating JIRA issue based on GitHub issue #%d", *issue.Number)
 
 	fields := jira.IssueFields{
 		Type: jira.IssueType{
-			Name: "Task", // TODO: Determine issue type
+			Name: "Aufgabe", // TODO: Determine issue type
```

Comment:
Seriously? 😂

Reply:
The todo is from the original codebase even 😅